### PR TITLE
Photon/various style fixes

### LINKED ISSUE #3333: Update search icon (including orientation) for in-content search field to match URL bar
Also see https://bugzilla.mozilla.org/show_bug.cgi?id=1396893
and https://bugzilla.mozilla.org/show_bug.cgi?id=1399642
- [ ] Change icon in the search field
- [ ] Make it so we detect platform and flip the icon the appropriate direction

https://mozilla.invisionapp.com/share/YNDB2H5SQ#/251492361_Search_Icon
Assets folder: https://drive.google.com/drive/folders/0B6F4iApRJzosVTdaZjN3UHY5NG8

### PR DESCRIPTION
**Pull request by k88hudson**
These are a few style/polish tweaks after a pass with the designers. Designers who requested/reviewed them are mentioned in the commit details.
Fix #3333.

### CODE REVIEW

**Comment by Mardak** [comment]
For reference before:
screen shot 2017-09-13 at 7 04 54 pm

After:
screen shot 2017-09-13 at 7 11 03 pm

File: system-addon/content-src/components/Search/_Search.scss [hunk 1]
```diff
@@ -3,14 +3,22 @@
   $search-height: 36px;
   $search-input-left-label-width: 35px;
   $search-button-width: 36px;
-  $search-glyph-image: url('#{$image-path}glyph-search-16.svg');
-  $search-glyph-size: 20px;
+  $search-glyph-image: url('chrome://browser/skin/find.svg');
```
Comment (Mardak) [inline system-addon/content-src/components/Search/_Search.scss]: Let's use `chrome://global/skin/icons/autocomplete-search.svg` to also fix #3333

**Comment by Mardak** [comment]
More reference before:
screen shot 2017-09-14 at 1 53 21 am

After:
screen shot 2017-09-14 at 1 51 23 am

File: system-addon/content-src/components/Search/_Search.scss (hunk 1 above)
Comment (Mardak) [reply system-addon/content-src/components/Search/_Search.scss]: Might need to double check the spacing/sizing.

**Comment by Mardak** [comment]
With find.svg:
image

With autocomplete-search.svg: 
image

But more interestingly on Linux find.svg:
image

Linux autocomplete-search.svg:
image

**Comment by Mardak** [comment]
With find.svg:
image

With autocomplete-search.svg: 
image

But more interestingly on Linux find.svg:
image

Linux autocomplete-search.svg:
image

@aaronrbenson

**Comment by aaronrbenson** [comment]
find.svg is the right icon. It turns out the icon used in the awesomebar is not updated with the right Photon icon. I've filed a separate bug for that.

**Comment by k88hudson** [comment]
@Mardak Yeah, that was the one we went with originally, but (visually) at least it's not right. I wonder it that file path will still be used though due to different builds for different OSs, I'll ask a question in the bug (https://bugzilla.mozilla.org/show_bug.cgi?id=1399642)

**Comment by as-pine-proxy** [comment]
[Treeherder: 80342d427e94742559a0f81abe9bfaeec7c5deed](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=80342d427e94742559a0f81abe9bfaeec7c5deed)

**Pull request merged**